src/components/field-input.tsx

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+
+import Label from "@/components/box-label";
+import Error from "@/components/error-msg";
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  label?: string;
+  placeholder?: string;
+  icon?: string;
+}
+
+/*
+This is a component for the HTML `<input>' tag
+Input:
+  label: The label displayed on the text box
+  placeholder: The placeholder message displayed on the text box
+  icon: An optional Image path to display an .svg file
+Output:
+  A input text box that is compatible w/ React-hook-forms
+*/
+export default function Field_Input<T extends FieldValues = FieldValues>({
+  label,
+  placeholder = "",
+  icon,
+  ...props
+}: FormProps<T>) {
+  const { field, fieldState } = useController(props);
+
+  !placeholder ? (placeholder = `Enter ${props.name}`) : "";
+  !label ? (label = props.name) : "";
```

Comment:
You've mutated your props which is a big nono, instead store them as a separate variable.
e.g.
`const computedLabel = label ?? props.name`

Reply:
yeah... sorry for not fixing that earlier. Should be fixed now